### PR TITLE
Updated of LabCorp Patient entry

### PR DESCRIPTION
**Pull request by andbenn**
LabCorp's Patient entry now supports 2FA.

### CODE REVIEW

File: entries/p/patient.labcorp.com.json [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-    },
+    "tfa": [
+      "sms",
+      "totp",
+      "custom-software"
+    ]
+    "custom-software": [
+      "Okta Verify"
+    ],
+    "documentation: "https://patient.labcorp.com/help-center/q-and-a/8000/2"
```
Comment (Carlgo11) [inline entries/p/patient.labcorp.com.json]: Thanks for the pull request @andbenn!

Just a small correction to fix the syntax errors :wink: 
```suggestion
    ],
    "custom-software": [
      "Okta Verify"
    ],
    "documentation: "https://patient.labcorp.com/help-center/q-and-a/8000/2",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by andbenn** [comment]
Updated.  The example I  was looking at wasn't complete.  Apologies.  Appears Ruby tests passed, not JavaScript or external tests.  Are those a concern?

File: entries/p/patient.labcorp.com.json [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
-    },
+    "tfa": [
+      "sms",
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Okta Verify"
+    ],
+    "documentation: "https://patient.labcorp.com/help-center/q-and-a/8000/2",
```
Comment (Carlgo11) [inline entries/p/patient.labcorp.com.json]: Found another syntax error 
```suggestion
    "documentation": "https://patient.labcorp.com/help-center/q-and-a/8000/2",
```

File: entries/p/patient.labcorp.com.json (hunk 1 above)
Comment (fletcherdodd) [inline entries/p/patient.labcorp.com.json]:
```suggestion
    "documentation": "https://patient.labcorp.com/help-center/q-and-a/8000/2",
```
This should fix the errors you're getting

File: entries/p/patient.labcorp.com.json (hunk 2 above)
Comment (fletcherdodd) [inline entries/p/patient.labcorp.com.json]:
```suggestion
    "documentation": "https://patient.labcorp.com/help-center/q-and-a/8000/2",
```
This should fix the error you're getting. :)

**Comment by andbenn** [comment]
Hello.  I'm not sure what I can do about the external tests that are blocking/failing.  What would the recommendation be?  I'm perplexed.

**Comment by hkamran80** [comment]
Feel free to correct me if I'm wrong, but IIRC Similarweb doesn't rank subdomains. We'll need to update the PR validator to fix that.

@Carlgo11 might be able to merge it anyway.

**Review by Carlgo11 — COMMENTED** [review]
The documentation link doesn't mention anything about the supported 2FA methods.
Could you please attach screenshots of the 2FA setup and/or login process where the different 2FA methods are visible.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment like this:
screenshot

**Comment by hkamran80** [comment]
> The documentation link doesn't mention anything about the supported 2FA methods.

This [page](https://patient.labcorp.com/help-center/q-and-a/8000/1) list the methods. I recommend changing the documentation URL to this one.

**Comment by andbenn** [comment]
Updated the URL to the one @hkamran80 recommended.  This should be ready now.  Thanks all.

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes, it takes longer to show up due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Comment by andbenn** [comment]
Thanks for the help making this change.  Hope I can do some more in the near future.  The site is a goldmine and bookmarked.

**Pull request merged**